### ISSUE TITLE
Hearing model | Remove hardcoded data

### ISSUE THREAD

**Issue by sharonwarner** [department-of-veterans-affairs/caseflow#9595]
We left 3 hardcoded methods 😧 

AC:
1) Update regional_office_name and regional_office_timezone to use the location generated by the AHL job
1) Update current_issue_count to count the number of request_issues

**Comment by oscarAtNava** [comment]
@sharonwarner The AHL job does not produce time zone information for the locations it provides as options. I think we still need to use the RO for time zone, or we need to add time zone to the AHL.

**Comment by sharonwarner** [comment]
Yes, we can use appeal.veteran_closest_regional_office for the RO.
